user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -96,16 +99,16 @@ def userPage(params) {
       }
     }
     if (settings."userCode${i}" && settings."userSlot${i}") {
-      def conflict = getConflicts(settings."userSlot${i}")
-      if (conflict.has_conflict) {
-        section("Conflicts:") {
-          theLocks.each { lock->
-            if (conflict."lock${lock.id}" && conflict."lock${lock.id}".conflicts != []) {
-              paragraph "${lock.displayName} slot ${fancyString(conflict."lock${lock.id}".conflicts)}"
-            }
-          }
-        }
-      }
+      //def conflict = getConflicts(settings."userSlot${i}")
+      //if (conflict.has_conflict) {
+      //  section("Conflicts:") {
+      //    theLocks.each { lock->
+      //      if (conflict."lock${lock.id}" && conflict."lock${lock.id}".conflicts != []) {
+      //        paragraph "${lock.displayName} slot ${fancyString(conflict."lock${lock.id}".conflicts)}"
+      //      }
+      //    }
+      //  }
+      //}
```

Comment:
This goes back to the error caused by getConflicts which prevented me from viewing user info in a second instance of the SmartApp.